Makefile

```diff
@@ -59,6 +59,8 @@ initial-promises: rudder-templates-cli.jar test
 	touch initial-promises/rudder_expected_reports.csv
 	# Provide a default rudder.json
 	cp variables.json initial-promises/node-server/rudder.json
+	# Provide a default rudder-parameters.json
+	cp rudder-parameters.json initial-promises/node-server/rudder-parameters.json
```

Comment:
rudder parameters are defined as `rudder.${namespaces}` but we need node properties here.